odl_video/models.py

```diff
@@ -0,0 +1,62 @@
+"""
+Classes related to models for ODL Video
+"""
+import datetime
+
+import pytz
+from django.db.models import DateTimeField, Manager, Model
+from django.db.models.query import QuerySet
+
+
+def now_in_utc():
+    """
+    Get the current time in UTC
+
+    Returns:
+        datetime.datetime: A datetime object for the current time
+    """
+    return datetime.datetime.now(tz=pytz.UTC)
+
+
+class TimestampedModelQuerySet(QuerySet):
+    """
+    Subclassed QuerySet for TimestampedModelManager
+    """
+    def update(self, **kwargs):
+        """
+        Automatically update updated_on timestamp when .update(). This is because .update()
+        does not go through .save(), thus will not auto_now, because it happens on the
+        database level without loading objects into memory.
+        """
+        if "updated_on" not in kwargs:
+            kwargs["updated_on"] = now_in_utc()
+        return super().update(**kwargs)
+
+
+class TimestampedModelManager(Manager):
+    """
+    Subclassed manager for TimestampedModel
+    """
+    def update(self, **kwargs):
```

Comment:
This method (and `TimestampedModelQuerySet.update`) don't seem to be covered by unit tests yet, can you add one to cover both when `updated_at` is and isn't in `kwargs`?  That should bring up the code coverage.